gcc/rust/hir/tree/rust-hir-expr.h

```diff
@@ -25,6 +25,7 @@
 #include "rust-hir-path.h"
 #include "rust-operators.h"
 #include "rust-expr.h"
+#include "tree.h"
```

Comment:
not required I think

Reply:
ahhh dumb lsp, i'll remove it

Reply:
happens to me all the time it's so frustrating lol. no worries